### PR TITLE
Allow AMA intakes when Veteran has no rated issues

### PR DESCRIPTION
**Pull request by leikkisa**
Connects #5868

This allows a Veteran without Ratings to still go through the intake process.  If they have no ratings, a message will be displayed to the user notifying them of this in case they need to add ratings.  Otherwise they can proceed with non-rated issues only.

Note: this does not create non-rated End Products for the non-rated issues yet.

### CODE REVIEW

**Comment by leikkisa** [comment]
Waiting to hear back if we want to remove the "Select from previous decision issues" header if there are no ratings.

screen shot 2018-07-30 at 3 45 50 pm

File: app/models/rating.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@ class Rating
 
   attr_accessor :participant_id, :profile_date, :promulgation_date
 
+  # One week buffer was added
```
Comment (leikkisa) [inline app/models/rating.rb]: This is not related to this PR, and I didn't put much because I couldn't remember the whole conversation about it (something to do with mail timing), but I thought it was useful to note that this is 1 year plus 1 week

File: app/models/rating.rb [hunk 2]
```diff
@@ -33,6 +34,8 @@ def fetch_issues
     [response[:rating_issues]].flatten.map do |issue_data|
       RatingIssue.from_bgs_hash(issue_data)
     end
+  rescue Savon::Error
```
Comment (kahlouie) [inline app/models/rating.rb]: Should we rescue a very specific Savon error or all of them? Not entirely sure what they look like, so I'd be curious if we always get the same error if they have no ratings.

File: app/models/rating.rb [hunk 3]
```diff
@@ -48,6 +51,8 @@ def fetch_timely(participant_id:, from_date:)
       end
 
       unsorted.sort_by(&:promulgation_date).reverse
+    rescue Savon::Error
```
Comment (kahlouie) [inline app/models/rating.rb]: here too.
